scripts/cbp_test.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+import salobj
+import SALPY_CBP
+import asyncio
+import logging
+import argh
+
+
+class CBPRemote():
+    def __init__(self):
+        self.remote = salobj.Remote(SALPY_CBP)
+        self.log = logging.getLogger(__name__)
+
+    async def standby(self):
+        standby_topic = self.remote.cmd_standby.DataType()
+        standby_ack = await self.remote.cmd_standby.start(standby_topic,timeout=10)
```

Comment:
We should discuss how we handle timeout lengths... I don't know what the answer is...